VeganVirus/VeganVirus/DesktopManager.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include <windows.h>
+#include <exception>
+#include <vector>
+#include <commctrl.h>
+#include <ShlObj.h>
+
+class DesktopManager
+{
+public:
+	DesktopManager();
+	~DesktopManager();
+
+	// method returns desktop icon position based on it's index
+	// input: icon index (int)
+	// return: icon position: (POINT)
```

Comment:
Remove second colon (`:`)

```suggestion
	// return: icon position (POINT)
```